packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-shell

```diff
@@ -41,7 +41,12 @@ do
             orig_filename="${2}"
 
             ARGS+=("${option}")
-            ARGS+=("$(resolve_path "${orig_filename}")")
+            # '-' means read file through STDIN
+            if [[ "${option}" = *file ]] && [ "${orig_filename}" = "-" ]; then
```

Comment:
Prefer `[[` everywhere so that we never have to stop and think about which one is being used.